recipe/meta.yaml

```
@@ -1,12 +1,12 @@
-{% set version = "0.29.33" %}
+{% set version = "3.0.0b1" %}
```

@h-vetinari should we give the latest beta a try?
```suggestion
{% set version = "3.0.0b3" %}
```

---

Thanks for keeping track of all the changes so far here!

---

You can already install 3.0.0b3 through conda-forge, though it needs a special label:
```
conda-forge -c conda-forge/label/cython_dev cython=3.0.0b3
```

This PR was mainly for running the test suite. I haven't gotten so much interest here, so I haven't kept up to date with all releases.

---

Yep I was just wondering if we wanted to see if the last couple of betas had addressed any of the open issues from your last post, or if there's some follow-up needed in subsequent Cython betas to add in whatever files were missing that caused the failures.